esp_serial_slave_link/idf_component.yml

```diff
@@ -0,0 +1,5 @@
+version: "0.1.0"
+description: "Espressifl Serial Slave Link Library"
+url: https://github.com/espressif/idf-extra-components/tree/master/esp_serial_slave_link
+dependencies:
+    idf: ">=4.3"
```

Comment:
I think component manager related workflow was not enabled for v4.3 case (e.g. `idf.py add-dependency`). Could you please check and then change this to `idf: ">=4.4"`?

Reply:
OK.. I just copied it from some other component..

Reply:
This seems not able to build with v4.4 for now. I set the version to v5.0. Then I may extend the support to v4.4 if necessary.